curriculum/challenges/english/25-front-end-development/review-html-accessibility/671a87a39b88245a579c2271.md

```diff
@@ -72,7 +72,7 @@ There are six main categories of ARIA roles:
 <button type="button" id="search-btn">Search</button>
 ```
 
-- **`aria-hidden` attribute**: Used to hide an element from people using assistive technology such as screen readers. For example, this can be used to hide decorative images that do not provide any meaningful content.
+- **`aria-hidden` attribute**: Used to hide an element from assistive technology such as screen readers. For example, this can be used to hide decorative images that do not provide any meaningful content.
```

Comment:
```suggestion
- **`aria-hidden` attribute**: Used to hide an element from assistive technologies such as screen readers. For example, this can be used to hide decorative images that do not provide any meaningful content.
```